components/brave_new_tab_ui/components/default/settings/index.ts

```diff
@@ -15,6 +15,8 @@ import favoritesUnselectedDark from './assets/favorites-unselected-dark.png'
 import frecencySelectedDark from './assets/frecency-selected-dark.png'
 import frecencyUnselectedDark from './assets/frecency-unselected-dark.png'
 
+import checkCircle from './assets/check-circle.png'
```

Comment:
Just confirming it's okay to use a PNG here? I think most of the time we prefer SVGs.